test/test.bundle.js

```diff
@@ -0,0 +1,395 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
I don't see why this file should be part of the repository?

Reply:
Good catch. Need to update .gitignore. 👍